core/controllers/domain_objects_validator_test.py

```diff
@@ -45,3 +47,79 @@ def test_correct_object_do_not_raises_exception(self):
             'property_name': 'title'
         }
         domain_objects_validator.validate_exploration_change(exploration_change)
+
+
+class ValidateNewConfigPropertyValuesTests(test_utils.GenericTestBase):
+    """Tests to validate config properties dict coming from API."""
+
+    def test_invalid_object_raises_exception(self):
+        config_properties = {'some_config_property': 20, }
+        with self.assertRaisesRegexp( # type: ignore[no-untyped-call]
+            Exception, 'some_config_property do not have any schema.'):
+            domain_objects_validator.validate_new_config_property_values(
+                config_properties)
+
+        config_properties = {1234: 20, }
+        with self.assertRaisesRegexp( # type: ignore[no-untyped-call]
+            Exception, 'config property name should be a string, received'
+            ': %s' % 1234):
+            domain_objects_validator.validate_new_config_property_values(
+                config_properties)
+
+    def test_valid_object_raises_no_exception(self):
+        config_properties = {
+            'max_number_of_tags_assigned_to_blog_post': 20,
+            }
+        domain_objects_validator.validate_new_config_property_values(
+            config_properties)
+
+
+class ValidateChangeDictForBlogPost(test_utils.GenericTestBase):
+    """Tests to validate change_dict coming from API."""
```

Comment:
Please update the docstring to include more details, because change_dict is too vague.

Reply:
Done.